### ISSUE TITLE
Consider shared location for suppressed checks across Validator and SirCAL

### ISSUE THREAD

**Issue by krivard** [cmu-delphi/covidcast-indicators#1194]
From the following [slack thread](https://delphi-org.slack.com/archives/C01LZ3A2UMU/p1628829182003000):

![image](https://user-images.githubusercontent.com/1158666/129574383-90897a94-5e8e-461c-b946-74a2d66f56a1.png)

> Nat DeFries
This is an intermittent issue caused by a period of low sample size. From July 11-31, increasing proportions of survey respondents were accidentally sent to an old survey version that did not include the item underlying this signal (FB-side report). Sample size has since recovered, but there was a period when no HRRs were reported due to low sample size (July 31-Aug 3).
What do we want to do here? Since this check is apparently looking at the last 7 days, can we just wait until it times out? (edited) 

> Katie
TL;DR: we just live with it for now.
Long version: The validation system has a much more sophisticated way of specifying suppressed checks than sirCAL does, but we can’t just drop sirCAL, since it serves as an additional check on the acquisition & replication system, which the validator can’t check. Ideally we move the list of suppressed checks to a centralized location and have both systems use the same information. However, that kind of thing is suspiciously similar to the anomalies record project we’re sponsoring with the MSE program this fall, so let’s flag this for re-consideration at midsemester.
